### ISSUE TITLE
Screen Reader Support and A11Y

### ISSUE THREAD

**Issue by robertlong** [matrix-org/thirdroom#399]
All the UI in Third Room uses the DOM on purpose so we can make everything accessible to screen readers. However, we haven't implemented everything yet. We still need to add ARIA tags on some things and ensure that keyboard navigation works everywhere.

There's also an opportunity to do more A11Y work in-world. We already have labels on interactable objects. We can make it easier to navigate to points of interest. We can also integrate the [EXT_mesh_features](https://github.com/CesiumGS/glTF/blob/proposal-EXT_mesh_features/extensions/2.0/Vendor/EXT_mesh_features/README.md) extension to make it easier to get more information about the world around you. There's a lot of work that can be done in this area.

When we're ready to tackle this, we should be working with the [XR Access Initiative](https://xraccess.org/) to make sure we're following the latest best practices for A11Y.
